nbextensions/usability/collapsible_headings/main.css

```diff
@@ -98,6 +98,18 @@ div.cell {
   border-width: 5px 2px 2px 4px;
 }
 
+
+.chb-start div:hover, .chb .chb-start.chb-hover, .jupyter-soft-selected .chb .chb-start {
+  border-top-width: 2px;
+  margin-top: 2px;
```

Comment:
you don't need to override `margin-top`, it's already set above. For simplicity and ease of maintenance, only override what you actually change.